addons/hitreactions/XEH_postInit.sqf

```diff
@@ -0,0 +1,65 @@
+#include "script_component.hpp"
+
+GVAR(armComponents) = [
+    "leftshoulder",
+    "rightshoulder",
+    "lefthand",
+    "leftforearm",
+    "leftarmroll",
+    "rightforearm",
+    "rightarmroll",
+    "righthand",
+    "rightarm"
+];
+
+GVAR(safePickupDistance) = DEFAULT_PICKUP_DISTANCE;
```

Comment:
Why not just use the define? Do you want that changeable on the fly? Should use `missionNamespace` specifically then.

Reply:
yes -- is missionnamespace not implied in func?